1449 improve examples (#1661)
open-contracting/standard: docs/guidance/map/amendments.md

# Updates and amendments

Information about a contracting (or planning) process often changes over time.

Each time information changes, a new OCDS release ought to be published. The new release can repeat information that was previously published, in addition to new and changed information.

> There are three types of change:

Reviewer: Changes are countable.

```suggestion
There are three types of changes:
```

* **New information**. For example, when information about the award of a contract is first released.
* **Updates to existing information**. For example, to correct errors in earlier releases, or to make minor adjustments to titles, descriptions or dates.
* **Amendments**. For example, when the value or duration of a contract is changed. The term amendment often has a specific legal meaning for a publisher. Certain changes to a tender, award or contract might only be permitted as part of an amendment.

The nature of a change can be made explicit using:

* **The release tag** field (`tag`), which is used to identify the type of change. For example, 'contract' identifies information about a new contract, 'contractUpdate' identifies an update to existing information about a contract, and 'contractAmendment' identifies a formal amendment to a contract.

> * **The amendments** fields (`tender.amendments` and `contract.amendments`), which are used to list amendments along with their rationales and references to the releases that contain before and after values.

Reviewer: Corrected fields, and added quotes for easier readability.

```suggestion
* **The amendments** fields (`tender.amendments`, `awards.amendments` and `contracts.amendments`), which are used to list amendments along with their rationales and references to the releases that contain "before" and "after" values.
```

## Worked examples

### Example 1: Tender updates and amendments

This example illustrates how new information, updates and amendments are modelled in OCDS.

#### Tender release

A buyer publishes an opportunity for the purchase of office supplies.

```{jsoninclude} ../../examples/amendments/tender.json
:jsonpointer: /records/0/releases/0
:expand: tender, tag
:title: Tender
```

#### Tender update release

> The buyer now indicates the opportunity's main procurement category. The new information is not a formal amendment so the publisher uses the 'tenderUpdate' tag and omits the `tender.amendments` field.

Reviewer: Comma before conjunction between full clauses.

```suggestion
The buyer now indicates the opportunity's main procurement category. The new information is not a formal amendment, so the publisher uses the 'tenderUpdate' tag and omits the `tender.amendments` field.
```

> The publisher chooses to repeat fields whose values are unchanged from the previous release. Such fields can be omitted when a publication provides access to historic releases.

Reviewer: Maybe make this an admonition, since access to historic releases is a bit tangential.

````suggestion
```{note}
The publisher chooses to repeat fields whose values are unchanged from the previous release. Such fields can be omitted when a publication provides access to historic releases.
```
````

Reviewer: Ah, I see you had commented around this change earlier:

> Whilst the proposed change is more correct in terms of describing the data, I think the intent of the second clause in the original sentence was to highlight that Microsoft and Siemens are separate legal entities so I suggest instead linking to the description of Organization.id which explains how that field relates to organisation identifiers.

I think Microsoft and Siemens are sufficiently well known, that readers know they are separate entities without drawing attention to the construction of their identifiers.

Reviewer 2: I think your second comment relates to https://github.com/open-contracting/standard/pull/1661/files#r1540402674, in which case I agree!

Reviewer: Indeed - I must have mis-scrolled before pasting!

```{jsoninclude} ../../examples/amendments/tender.json
:jsonpointer: /records/0/releases/1
:expand: tender, tag
:title: TenderUpdate
```

#### Tender amendment release

> The buyer increases the estimated value of the opportunity. This change is a formal amendment so the publisher uses the 'tenderAmendment' tag and populates the `tender.amendments` field.

Reviewer: ```suggestion
The buyer increases the estimated value of the opportunity. This change is a formal amendment, so the publisher uses the 'tenderAmendment' tag and populates the `tender.amendments` field.
```

#### Record

`releases` contains the above releases, `compiledRelease` contains the latest value of each field, and `versionedRelease` contains a history of changes to each field.

> The `releaseID` and `amendsReleaseID` fields in the `amendments` array of the compiled release can be looked up in `releases` and `versionedRelease` to identify what changed.

Reviewer: ```suggestion
The `releaseID` and `amendsReleaseID` fields in the `amendments` array of the compiled release can be looked up in `releases` and `versionedRelease` to determine what changed.
```

```{jsoninclude} ../../examples/amendments/tender.json
:jsonpointer: /records/0
:expand: compiledRelease, versionedRelease, value, amount
:title: FullRecord
```

```{hint}
[Download](../../examples/amendments/tender.json) the record example and use the [Data Review Tool](https://review.standard.open-contracting.org/) to explore the changes in the contracting process.
```

> ```{admonition} Contract updates and amendments

Reviewer: ```suggestion
```{admonition} Award and contract updates and amendments
```

> Contract updates and amendments are modelled in the same way: the 'contract', 'contractUpdate' and 'contractAmendment' release tags distinguish the type of change and amendments are listed in the `contract.amendments` field.

Reviewer: ```suggestion
Award and contract updates and amendments are modelled in the same way. The 'award', 'contract', 'awardUpdate', 'contractUpdate' and 'contractAmendment' release tags indicate the type of change. Amendments are listed in the `awards.amendments` and `contracts.amendments` fields.
```
